### DISCUSSION TITLE
Runtime-First FPGA Interchange Routing Contest @ FPGA’24

### DISCUSSION THREAD

**Discussion by clavin-xlnx** [Xilinx/RapidWright#833] [Announcements]
The RapidWright team is sponsoring a runtime-optimized FPGA routing competition featuring the FPGA Interchange Format at [ISFPGA 2024](https://www.isfpga.org/fpga-routing-contest/)!  The registration deadline to enter is October 20th.  The competition will have cash prizes 💰🥇 and possible hardware gifts 🎁 for participants! 

See the official announcement for full details here: [Runtime-First FPGA Interchange Routing Contest](https://xilinx.github.io/fpga24_routing_contest/).

**Comment by eddieh-xlnx** [comment]
ℹ️ Registration deadline has been extended to November 20th!
